### DISCUSSION TITLE
1.0.9

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#38] [Обновления]
## What's Changed
* Fix translation request by @mynovelhost in https://github.com/ilyhalight/voice-over-translation/pull/30
* Force remove sec-ch-ua* headers by @mynovelhost in https://github.com/ilyhalight/voice-over-translation/pull/31
* improve(styles): button by @riddle418 in https://github.com/ilyhalight/voice-over-translation/pull/32

## New Contributors
* @riddle418 made their first contribution in https://github.com/ilyhalight/voice-over-translation/pull/32

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.0.8...1.0.9

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.0.9'>1.0.9</a>.</em>

**Comment by FiLinX** [comment]
обновил скрипт в темпермонке щас, но что-то непойму, ведь вроде указано что применён фикс https://github.com/ilyhalight/voice-over-translation/pull/30 
а прям вот из этого фикса примеры ссылок на twitch\vimeo\9gag\vk  -  панельки перевода не обнаруживают(
может что-то ещё требуется?

**Reply by ilyhalight** [reply]
Это гитхаб подхватил старые изменения, в которых был только предоставлен небольшой вариант, как примерно работает функция перевода для vimeo, gag и т.п.

**Reply by ilyhalight** [reply]
Саму кнопку на эти сайты ещё не добавил. Это обновление больше для того, чтобы зафиксировать стабильную версию и добавить мелкие функции, которые давно просили. (что добавлено/изменено/исправлено можно прочитать тут: https://github.com/ilyhalight/voice-over-translation/blob/master/changelog.md)
